Makefile

```
@@ -85,11 +86,12 @@ container: licenses
 container-test: sdk-image licenses
 	docker build \
 		--network=host \
-		--build-arg GO_LDFLAGS='$(GO_LDFLAGS)' \
-		--build-arg GOARCH='$(GOARCH)' \
-		--build-arg SHORT_SHA='$(SHORT_SHA)' \
-		--build-arg NOCACHE='$(shell date +"%s")' \
-		--build-arg LICENSES_IMAGE=$(LICENSES_IMAGE) \
+		--build-arg GO_LDFLAGS \
+		--build-arg GOARCH \
+		--build-arg "SHORT_SHA=$(SHORT_SHA)" \
```

nit: Can we do the same (setting `build-arg` without a value and just use local environment variable) for `SHORT_SHA` ( and `LICENSES_IMAGE` below)?
```suggestion
		--build-arg GOARCH \
		--build-arg SHORT_SHA \
```

---

We can't as is, the variable [`GOARCH` is exported](https://github.com/bottlerocket-os/bottlerocket-update-operator/pull/21/files/e4d254471a6ec2d920a471bb29aa75025154a6ee#diff-b67911656ef5d18c4ae36cb6741b7965R52) whereas [`SHORT_SHA` is not](https://github.com/bottlerocket-os/bottlerocket-update-operator/pull/21/files/e4d254471a6ec2d920a471bb29aa75025154a6ee#diff-b67911656ef5d18c4ae36cb6741b7965R20) - to use the nicer `--buildarg MY_ENV_VAR` syntax the variable must be present. There's only a couple variables that are `export`ed that are related to build-wide influencing variables (like `GOARCH`). I would prefer to not have `SHORT_SHA` also exported too because in my mind it doesn't quite fit in with that sort of influential build/compile environment variable.

Do you think it would be less tempting if we instead made all the `--build-arg` flags use the fuller syntax? (ie: `--build-arg MY_ENV_VAR=my-value`)

---

I would prefer the syntax to be consistent. But it is not a hard blocker for me.

---

I agree, I'll update them to be consistent so we aren't tempted to get them any more out of whack with each other :+1: